File: New/python/strings.py
def count_vowels(text):
    vowels_letters = ['a', 'e', 'i', 'o', 'u']
    count = 0
    for letter in text:
        if letter.lower() in vowels_letters:
            count += 1
    return count

#def is_palindrom(text):
    #Всё ещё не знаю как реализовать, помоги. Не знаю как проверять равенство букв

def count_letters(text):
    letters = {}
    count = 1
    for letter in text:
        if letter not in letters:
            letters[letter] = count
        else:
            letters[letter] += 1
    return letters

def reverse_word(text):
    splitted_text = text.split()
    #Не знаю как переставлять слова, объясни

vowel = count_vowels("hello world!")
word = count_letters("banaaaana")
print(vowel)
print(word)

word1 = reverse_word("hello world python")
print(word1)

File: New/js/strings.js
function countLetters(text) {
    let counter = 0;
    for(let letter of text) {
        counter++;
    }
    return counter;
}

function countSpaces(text) {
    let counter = 0;
    for(let space of text) {
        if(space === " ") {
            counter++;
        }
    }
    return counter;
}

function capitalize(text) {
    text.replace(text[0], text[0].toUpper());
    return text;
    //Не знаю как реализовать, объясни
}

function replaceSpaces(text) {
    newSpace = text.replace(" ", "_");
    return newSpace;
}

function containsWord(text, word) {
    splittedText = text.split();
    if (word in splittedText) {
        return true;
    } else {
        return false;
    }
}

word = countLetters("bear");
console.log(`Letters count: ${word}`);
space_count = countSpaces("Hello world dude!");
console.log(`Spaces count: ${space_count}`);
changeSpace = replaceSpaces("Hello world!");
console.log(changeSpace);
checkWord1 = containsWord("Hello world!", "Hello");
checkWord2 = containsWord("Hello world!", "python");
console.log(`First word: ${checkWord1}`);
console.log(`Second word: ${checkWord2}`);
//newWord = capitalize("program");
//console.log(newWord);

//Не знаю как сделать реверс, а в поиске слова в тексте, я знаю как сделать, но как бы не пытался не работает, помоги пожалуйста
